17.cpp

```diff
@@ -0,0 +1,50 @@
+// https://leetcode-cn.com/problems/letter-combinations-of-a-phone-number/
+#include <iostream>
+#include <string>
+#include <vector>
+std::vector<std::string> res;
```

Comment:
Don't use global vars.

Reply:
I want save memory and get a higher score